ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/eos_cli.yml

```diff
@@ -2,6 +2,10 @@
 # Testing the rendering of `eos_cli` when this knob is true
 avd_eos_designs_enforce_duplication_checks_across_all_models: true
 
+platform_settings:
```

Comment:
why is it in this file? Please move the test to some hosts where it makes sense. Here you are not even using the defined platform so I guess you can just move this to another place with `platform_settings`?

Reply:
Added a comment